<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Completed Tasks</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/tick.png</normaloff>../resources/tick.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1085</width>
      <height>650</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1085</width>
       <height>650</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1085</width>
        <height>650</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../resources/bg.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>451</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="title_2">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../resources/finished_assignments.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="complete_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>751</width>
        <height>491</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    background-color: black;
    color: white;
    selection-background-color: #007BFF; /* Highlight color when a row is selected */
}

QHeaderView::section {
    background-color: #007BFF; /* Header background color */
    color: white;
    padding: 4px;
    border: 1px solid #666666;
}

QTableWidget QTableCornerButton::section {
    background-color: #007BFF; /* Top-left corner background color */
    border: 1px solid #666666;
}

QScrollBar:vertical {
    border: 1px solid #666666;
    background: black;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #007BFF;
    min-height: 0px;
}

QScrollBar::add-line:vertical {
    background: black;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    background: black;
    height: 0 px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}
</string>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>240</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Completed Tasks</string>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time Of Completion</string>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Of Completition</string>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>170</y>
        <width>241</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="refresh">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid black;
    border-radius: 25px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 255, 50);
    border: 2px solid #000080;
    transition: background-color 0.3s, border 0.3s;
}</string>
         </property>
         <property name="text">
          <string>  Refresh</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../resources/refresh.png</normaloff>../resources/refresh.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undo">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid black;
    border-radius: 25px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 255, 50);
    border: 2px solid #000080;
    transition: background-color 0.3s, border 0.3s;
}</string>
         </property>
         <property name="text">
          <string> Undo Clear</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../resources/undo.png</normaloff>../resources/undo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid black;
    border-radius: 25px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 255, 50);
    border: 2px solid #000080;
    transition: background-color 0.3s, border 0.3s;
}</string>
         </property>
         <property name="text">
          <string>  Clear List</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../resources/clear.png</normaloff>../resources/clear.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
